apps/ui/src/views/Space/Editor.vue

```diff
@@ -63,6 +63,10 @@ const modalOpenTerms = ref(false);
 const previewEnabled = ref(false);
 const sending = ref(false);
 const enforcedVoteType = ref<VoteType | null>(null);
+const customProposalTime = reactive<{
+  start?: number;
+  minEnd?: number;
+}>({});
```

Comment:
I'd prefer to use `ref`s instead of `reactive` due to [its limitations](https://vuejs.org/guide/essentials/reactivity-fundamentals.html#limitations-of-reactive) and sometimes ending up in annoying situations due to those.